tests/network-tests/src/fixtures/membership/VerifyValidatorAccountFixture.ts

```diff
@@ -2,48 +2,91 @@ import { assert } from 'chai'
 import Long from 'long'
 import { SubmittableExtrinsic } from '@polkadot/api/types'
 import { ISubmittableResult } from '@polkadot/types/types/'
-import { WorkerId, ForumPostId } from '@joystream/types/primitives'
 import { RemarkMetadataAction } from '@joystream/metadata-protobuf'
 import { Api } from '../../Api'
 import { QueryNodeApi } from '../../QueryNodeApi'
 import { EventDetails } from '../../types'
 import { Utils } from '../../utils'
 import { BaseQueryNodeFixture } from '../../Fixture'
+import {
+  MembershipFieldsFragment,
+  MemberVerificationStatusUpdatedEventFieldsFragment,
+} from 'src/graphql/generated/queries'
+import { createType } from '@joystream/types'
 
 export type ValidaotrAccountInput = {
   memberId: string
-  isVerified: boolean
-  asWorker?: string
+  isVerifiedValidator: boolean
 }
 
-export class VerifyValidatorAccountFixture extends BaseQueryNodeFixture {
+export class VerifyValidatorProfileFixture extends BaseQueryNodeFixture {
   protected verifyValidator: ValidaotrAccountInput[]
+  private tx?: SubmittableExtrinsic<'promise'>
 
   public constructor(api: Api, query: QueryNodeApi, verifyValidator: ValidaotrAccountInput[]) {
     super(api, query)
     this.verifyValidator = verifyValidator
   }
 
   protected async getEventFromResult(result: ISubmittableResult): Promise<EventDetails> {
-    if (this.api.findEvent(result, 'operationsWorkingGroupBeta', 'WorkerRemarked')) {
-      return this.api.getEventDetails(result, 'operationsWorkingGroupBeta', 'WorkerRemarked')
+    if (this.api.findEvent(result, 'membershipWorkingGroup', 'WorkerRemarked')) {
+      return this.api.getEventDetails(result, 'membershipWorkingGroup', 'WorkerRemarked')
     } else {
-      return this.api.getEventDetails(result, 'operationsWorkingGroupBeta', 'LeadRemarked')
+      return this.api.getEventDetails(result, 'membershipWorkingGroup', 'LeadRemarked')
     }
   }
 
   protected async getExtrinsics(): Promise<SubmittableExtrinsic<'promise'>[]> {
     return this.verifyValidator.map((u) => {
       const metadata = Utils.metadataToBytes(RemarkMetadataAction, {
-        verifyValidator: { memberId: Long.fromString(String(u.memberId)), isVerified: u.isVerified },
+        verifyValidator: { memberId: Long.fromString(String(u.memberId)), isVerified: u.isVerifiedValidator },
       })
-      return u.asWorker
-        ? this.api.tx.operationsWorkingGroupBeta.workerRemark(u.asWorker, metadata)
-        : this.api.tx.operationsWorkingGroupBeta.leadRemark(metadata)
+      return u.memberId
+        ? this.api.tx.membershipWorkingGroup.workerRemark(u.memberId, metadata)
+        : this.api.tx.membershipWorkingGroup.leadRemark(metadata)
     })
   }
 
+  private assetVerifyValidatorTest(qMember: MembershipFieldsFragment[] | null): void {
```

Comment:
Please rename this method to:
```suggestion
  private assertQueriedMembershipAreValid(qMember: MembershipFieldsFragment[] | null): void {
```